x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/product_selector/product_selector.test.tsx

```diff
@@ -25,7 +25,7 @@ describe('ProductSelector', () => {
     setMockValues({ config: { host: '' } });
     const wrapper = shallow(<ProductSelector access={{}} />);
 
-    expect(wrapper.find(EuiPage).hasClass('enterpriseSearchOverview')).toBe(true);
+    expect(wrapper.find(KibanaPageTemplate).hasClass('enterpriseSearchOverview')).toBe(true);
```

Comment:
You can likely just get rid of this assertion (+ import) TBH, I don't think it's useful now that we don't have a custom class

```suggestion
```